language/test/Language_test.rb

```diff
@@ -0,0 +1,151 @@
+#!/usr/bin/env rspec
+
+require_relative "test_helper"
+
+Yast.import "Mode"
+Yast::Mode.SetMode("normal")
```

Comment:
This should not be needed, you call `allow(Yast::Mode).to receive(:mode).and_return("normal")` in a test below.